packages/resend/src/index.ts

```diff
@@ -0,0 +1,108 @@
+import { Resend } from 'resend';
+import { type ReactNode } from 'react';
+
+import { type EmailProvider } from '@modelence/types';
+import { getConfig } from 'modelence/server';
+
+// types are duplicated for typedoc
+export type EmailAttachment = {
+  filename: string;
+  content: Buffer | string;
+  contentType: string;
+};
+
+export type EmailPayload = {
+  from: string;
+  to: string;
+  subject: string;
+  html?: string;
+  text?: string;
+  react?: ReactNode;
+  cc?: string;
+  bcc?: string;
```

Comment:
What about multiple items here? Are they just going to be comma separated? Overall, I think it's ok to just mirror what Resend and others do because we don't want to reinvent the interface, but just curious.